dbms/src/Flash/Executor/PipelineExecutor.cpp

```diff
@@ -108,7 +108,7 @@ ExecutionResult PipelineExecutor::execute(ResultHandler && result_handler)
         scheduleEvents();
         wait();
     }
-    LOG_TRACE(log, "query finish with {}", exec_context.getQueryProfileInfo().toJson());
+    LOG_DEBUG(log, "query finish with {}", exec_context.getQueryProfileInfo().toJson());
```

Comment:
may be info level? @windtalker 
```suggestion
    LOG_INFO(log, "query finish with {}", exec_context.getQueryProfileInfo().toJson());
```

Reply:
LGTM